Makefile

```diff
@@ -60,6 +60,7 @@ lint:
 	flake8
 	isort --check-only .
```

Comment:
Even if we wouldn't want to immediately move away from flake8 as requested by @real-yfprojects, I think we can safely move from isort to ruff.

Reply:
Having https://github.com/borgbase/vorta/pull/1666#discussion_r1143728743 in mind I suggest keeping this as a check.